crates/trycmd/src/runner.rs

```diff
@@ -68,10 +69,11 @@ impl Runner {
                                         status.spawn.status.summary(),
                                     );
                                     if let Some(duration) = status.duration {
+                                        let duration = SignedDuration::from_nanos(duration.as_nanos() as i64);
```

Comment:
Is `status.duration` a `std::time::Duration`? If so, consider `SignedDuration::try_from(duration).unwrap()`, which will have an explicit panic instead of a silent truncation.

Reply:
> I was not sure if we assumed panic was allowed in this context, but I see now it's already used at various places
Just updated the PR with your code (and thanks for jiff 😄)

Think of `unwrap` as an assert with such a clear intention that it doesn't need more clarification.

There are different ways of thinking of this
- If a test were to take longer than `i64` can represent, that is highly unusual and a panic isn't too bad.  In that case, we should document that with an `.expect()`
- If a test takes longer than a `i64` can represent, we should still report results and it would be fine to leave the value at `i64::MAX`
- If a test takes longer than a `i64` can represent, we should still report results and we should indicate that the test too longer than we could track

Reply:
This was marked as resolved yet `unwrap()` is still being used

Reply:
My bad, i will go back to this